Dockerfile.ubuntu

```diff
@@ -142,9 +143,12 @@ ENV TEST_LOG="/root/.cache/bazel/_bazel_root/bc57d4817a53cab8c785464da57d1983/ex
 
 ################### BUILD OPENVINO FROM SOURCE - buildarg ov_use_binary=0  ############################
 # Build OpenVINO and nGraph (OV dependency) with D_GLIBCXX_USE_CXX11_ABI=0 or 1
-RUN if [ "$ov_use_binary" == "0" ] ; then true ; else exit 0 ; fi ; git clone https://github.com/openvinotoolkit/openvino --recurse-submodules --branch $ov_source_branch --depth 1 /openvino
+RUN if [ "$sentencepiece" == "1" ] ; then true ; else exit 0 ; fi ; git clone https://github.com/openvinotoolkit/openvino_contrib.git --recurse-submodules --branch $ov_contrib_branch --depth 1 /openvino_contrib
+RUN if [ "$sentencepiece" == "1" ] ; then true ; else exit 0 ; fi ; git clone https://github.com/slyalin/openvino.git --recurse-submodules --branch muse_leftovers --depth 1 /openvino
+RUN if [ "$ov_use_binary" == "0" ] ; then true ; else exit 0 ; fi ; if [ "$sentencepiece" == "0" ] ; then true ; else exit 0 ; fi ; git clone https://github.com/openvinotoolkit/openvino --recurse-submodules --branch $ov_source_branch --depth 1 /openvino
 WORKDIR /openvino/build
-RUN if [ "$ov_use_binary" == "0" ] ; then true ; else exit 0 ; fi ; cmake -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE -DENABLE_SAMPLES=0 -DNGRAPH_USE_CXX_ABI=1 -DCMAKE_CXX_FLAGS=" -D_GLIBCXX_USE_CXX11_ABI=1 -Wno-error=parentheses "  ..
+RUN if [ "$sentencepiece" == "1" ] ; then true ; else exit 0 ; fi ; cmake -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE -DOPENVINO_EXTRA_MODULES=/openvino_contrib/modules -DBUILD_nvidia_plugin=OFF -DBUILD_arm_plugin=OFF -DBUILD_mo_pytorch=OFF -DBUILD_optimum=OFF -DENABLE_SAMPLES=0 -DNGRAPH_USE_CXX_ABI=1 -DCMAKE_CXX_FLAGS=" -D_GLIBCXX_USE_CXX11_ABI=1 -Wno-error=parentheses "  ..
```

Comment:
it is building all cpu extensions, should we filer it or change the arg name to custom_ops ?